<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../actiona.qrc">
    <normaloff>:/icons/options.png</normaloff>:/icons/options.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="settingsTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Loading window:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="showLoadingWindow">
         <property name="text">
          <string>Show the loading window</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Taskbar icon:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="showTaskbarIcon">
           <property name="text">
            <string>Show the taskbar icon</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noSysTrayLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/icons/warning.png&quot; /&gt;  &lt;span style=&quot; font-size:8pt;&quot;&gt;Unable to setup the system a tray icon.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noSysTrayMessagesLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/icons/warning.png&quot; /&gt;  System tray messages are not available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="showWindowAfterExecutionLabel">
         <property name="text">
          <string>Show window after execution:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="showWindowAfterExecution">
         <property name="text">
          <string>Set the main window as the foreground window after execution</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Console start/end separators:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="addStartEndSeparators">
         <property name="text">
          <string>Add start/end separators</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Last opened script:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="reopenLastScript">
         <property name="text">
          <string>Reopen the last opened script</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Maximum recent files:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="maxRecentFiles">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="fileAssociationsLabel">
         <property name="text">
          <string>File associations:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="associateASCRCheckBox">
           <property name="text">
            <string>Associate .ascr files with Actiona</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="associateACODCheckBox">
           <property name="text">
            <string>Associate .acod files with Actiona</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="languageComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Action dialogs:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="preloadActionDialogsCheckBox">
         <property name="text">
          <string>Preload action dialogs at startup</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Theme:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="themeComboBox">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dark</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Console maximum entries:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="consoleMaxEntries">
         <property name="specialValueText">
          <string>unlimited</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="actionsTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/sfx.png</normaloff>:/images/sfx.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Execution window:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="executionWindowGroup">
           <property name="title">
            <string>Execution window</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ActionTools::ScreenPositionWidget" name="executionWindowPosition" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Console window:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="consoleWindowGroup">
           <property name="title">
            <string>Console window</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ActionTools::ScreenPositionWidget" name="consoleWindowPosition" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Start/Stop execution hotkey:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SettingsKeyEdit" name="stopExecutionHotkey"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Pause/Resume execution hotkey:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="SettingsKeyEdit" name="pauseExecutionHotkey"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Switch text/code key:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="SettingsKeyEdit" name="switchTextCode"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Open editor key:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="SettingsKeyEdit" name="openEditorKey"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Code syntax:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="checkCodeSyntaxAutomatically">
         <property name="text">
          <string>Check code syntax automatically</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Heatmap colors:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Minimum:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ColorPickerPushButton" name="heatmapMinColorPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Maximum:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ColorPickerPushButton" name="heatmapMaxColorPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="networkTab">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="updatesCheckLabel">
         <property name="text">
          <string>Check for updates:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="updatesCheck">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Once a day</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Once a week</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Once a month</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Proxy:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="noProxyRadioButton">
           <property name="text">
            <string>Do not use a proxy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="systemProxyRadioButton">
           <property name="text">
            <string>Use the system proxy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="customProxyRadioButton">
           <property name="text">
            <string>Use custom proxy settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="proxySettings">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Proxy settings</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Host :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="proxyHost"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="proxyPort"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Port :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>User :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Password :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="proxyUser"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="proxyPassword"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Proxy type :</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="proxyType">
              <item>
               <property name="text">
                <string>Http</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Socks5</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="testConnectivity">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Test connectivity</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsKeyEdit</class>
   <extends>QLineEdit</extends>
   <header>settingskeyedit.hpp</header>
  </customwidget>
  <customwidget>
   <class>ActionTools::ScreenPositionWidget</class>
   <extends>QWidget</extends>
   <header location="global">actiontools/screenpositionwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPickerPushButton</class>
   <extends>QPushButton</extends>
   <header>colorpickerpushbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../actiona.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
